kafka.go

```diff
@@ -14,148 +14,254 @@
  * limitations under the License.
  */
 
-package eventpublisher
+package eventstream
 
 import (
+	"context"
 	"encoding/json"
-	"time"
+	"fmt"
+	"sync"
 
-	"github.com/Shopify/sarama"
 	"github.com/sirupsen/logrus"
+
+	"time"
+
+	"github.com/cenkalti/backoff"
+	"github.com/segmentio/kafka-go"
```

Comment:
the imports are still not sorted properly

Reply:
sure, will update it
thanks